collectors/charts.d.plugin/apcupsd/apcupsd.chart.sh

```diff
@@ -41,13 +41,13 @@ apcupsd_check() {
 
 	local host working=0 failed=0
 	for host in "${!apcupsd_sources[@]}"; do
-		run apcupsd_get "${apcupsd_sources[${host}]}" >/dev/null
+		apcupsd_get "${apcupsd_sources[${host}]}" >/dev/null
```

Comment:
you mention passing correct info to run, but here you remove it.
Could you elaborate a bit on the change? Thanks!

Reply:
Line# 26 would cause run() to run twice. I thought that was the problem.
Line# 44 was passing `run apcupsd_get` back into `run`
I think this would essentially send the array apcupsd_sources through twice.
That seemed like a problem.

Reply:
Could you provide details on the PR description around the problem you faced and what are you trying to fix with this change?

Reply:
Done. Let me know if you need me to provide anything else.

Reply:
Ok now i see what's the case. Just go to file `charts.d.plugin.in` line 307 and do the following:

change this
```
307         rm "${TMP_DIR}/run.${pid}"
```
to this
```
307         rm -f "${TMP_DIR}/run.${pid}"
```

I am not 100% sure why this doesn't get created all the times even empty, but adding -f should silence this soft error

Reply:
That would fix it. I was afraid to do something like that in case it was problematic that the file wasn't being created in the first place. I'll add a new commit with this change if you are good with it.

Reply:
yes, please go ahead.